hathor/sysctl/init_file_loader.py

```diff
@@ -0,0 +1,21 @@
+from typing import Optional
+
+from hathor.sysctl.runner import SysctlRunner
+
+
+class SysctlInitFileLoader:
+    def __init__(self, runner: SysctlRunner, init_file: Optional[str]) -> None:
```

Comment:
I think `init_file` must be a string (instead of an optional string).

Reply:
Done